(lack of) protogen-grpc tool fails make generate command

## Describe the bug
when building opencloud from source (since 7.0.0-rc0?), the `make generate` command tries to run `protoc-gen-go-grpc`.
This fails with
```
generating protobuf content
cd ../../protogen/proto && /root/go/bin/buf-v1.39.0 generate
Failure: plugin go-grpc: exec: "../../.bingo/protoc-gen-go-grpc": stat ../../.bingo/protoc-gen-go-grpc: no such file or directory
make[1]: Leaving directory '/workspace/erik/apkbuilds/apkbuilds/opencloud/src/opencloud-7.0.0/services/settings'
make[1]: *** [../../.make/protobuf.mk:23: /workspace/erik/apkbuilds/apkbuilds/opencloud/src/opencloud-7.0.0/protogen/buf.sha1.lock] Error 1
make: *** [Makefile:141: generate-prod] Error 1
>>> ERROR: opencloud: build failed
```

## Steps to reproduce
1. clone repository
2. run `make generate`
3. observe the build failure

## Expected behavior
The generation step should install the tool and run it

## Actual behavior
The tool is not installed and the build returns code 1

## Additional context
I am currently using this patch to work around this:
```patch
--- a/.make/protobuf.mk
+++ b/.make/protobuf.mk
@@ -11,6 +11,7 @@ protoc-deps: $(BINGO)
     @cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/owncloud/protoc-gen-microweb
     @cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
     @cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/favadi/protoc-go-inject-tag
+    @cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l google.golang.org/grpc/cmd/protoc-gen-go-grpc
 
```
for some reason this does not fail the build pipeline in this repo:
[example](https://ci.opencloud.rocks/repos/3/pipeline/101/17)

<img width="1369" height="301" alt="Image" src="https://github.com/user-attachments/assets/1715768f-e19c-4323-ada0-8c3743fe8556" />

--- a/.bingo/Variables.mk
+++ b/.bingo/Variables.mk
@@ -83,11 +83,11 @@ $(PIGEON): $(BINGO_DIR)/pigeon.mod
 	@echo "(re)installing $(GOBIN)/pigeon-v1.3.0"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=pigeon.mod -o=$(GOBIN)/pigeon-v1.3.0 "github.com/mna/pigeon"
 
-PROTOC_GEN_GO_GRPC := $(GOBIN)/protoc-gen-go-grpc-v1.81.0
+PROTOC_GEN_GO_GRPC := $(GOBIN)/protoc-gen-go-grpc-v1.6.2
 $(PROTOC_GEN_GO_GRPC): $(BINGO_DIR)/protoc-gen-go-grpc.mod
 	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
-	@echo "(re)installing $(GOBIN)/protoc-gen-go-grpc-v1.81.0"
-	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=protoc-gen-go-grpc.mod -o=$(GOBIN)/protoc-gen-go-grpc-v1.81.0 "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
+	@echo "(re)installing $(GOBIN)/protoc-gen-go-grpc-v1.6.2"
+	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=protoc-gen-go-grpc.mod -o=$(GOBIN)/protoc-gen-go-grpc-v1.6.2 "google.golang.org/grpc/cmd/protoc-gen-go-grpc"
 
 PROTOC_GEN_GO := $(GOBIN)/protoc-gen-go-v1.28.1
 $(PROTOC_GEN_GO): $(BINGO_DIR)/protoc-gen-go.mod

--- a/.bingo/variables.env
+++ b/.bingo/variables.env
@@ -30,7 +30,7 @@ MUTAGEN="${GOBIN}/mutagen-v0.18.1"
 
 PIGEON="${GOBIN}/pigeon-v1.3.0"
 
-PROTOC_GEN_GO_GRPC="${GOBIN}/protoc-gen-go-grpc-v1.81.0"
+PROTOC_GEN_GO_GRPC="${GOBIN}/protoc-gen-go-grpc-v1.6.2"
 
 PROTOC_GEN_GO="${GOBIN}/protoc-gen-go-v1.28.1"
 

--- a/.make/protobuf.mk
+++ b/.make/protobuf.mk
@@ -11,6 +11,7 @@ protoc-deps: $(BINGO)
 	@cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/owncloud/protoc-gen-microweb
 	@cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
 	@cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l github.com/favadi/protoc-go-inject-tag
+	@cd ../.. && GOPATH="" GOBIN=".bingo" $(BINGO) get -l google.golang.org/grpc/cmd/protoc-gen-go-grpc
 
 .PHONY: buf-generate
 buf-generate: $(SHA1_LOCK_FILE)

